Examples/ServiceExamples/Scripts/ExampleStreaming.cs

```diff
@@ -177,7 +177,7 @@ private IEnumerator RecordingHandler()
 
                 //  Create AudioData and use the samples we just created
                 AudioData record = new AudioData();
-                record.MaxLevel = Mathf.Max(samples);
+                record.MaxLevel = Mathf.Abs(Mathf.Max(samples));
```

Comment:
I think @cloudconstable suggested this will only get highest positive values and was worried if we have a sample that is completely negative. I don't know enough about audio to understand what the samples look like but heres a fix for that
```
record.MaxLevel = Mathf.Max(Mathf.Abs(Mathf.Min(samples)), Mathf.Max(samples));
```

Reply:
Of course! Good catch, I'll change this.